lib/apis/fetch/logger.js

```diff
@@ -0,0 +1,23 @@
+const { URL } = require("url")
+
+export default url => {
+  const cache = global.requestCount
```

Comment:
It's good to avoid globals at all costs—in this case hoisting this up to the top of the module should suffice, e.g.

````javascript
const { URL } = require("url")

const requestCount = {}

export default url => {
  // ...
}
````

Reply:
oh yeah, good point, it's all in the same file now